changelog-master.xml

```diff
@@ -5,4 +5,6 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
 <include  file="sql/views.sql" relativeToChangelogFile="true"/>
```

Comment:
Hey, I am not it is painful, do we need it at all? `Develop` has one entry in `EDH_WMS_STAGING.DATABASECHANGELOG` I could just delete it so Liquibase would not complain - `003_views.sql` supersedes it anyway. I could do it in `develop`, I've checked `integration` and my privileges got messed up, but someone with privs could do it.

Reply:
I didn't want to go messing with the liquibase table. Felt like messing with the internals of the tool.  Then there has to be manual steps in the RN and it makes all that more complex and likely to go wrong.

Reply:
`DATABASECHANGELOG` is a registry Liquibase relies on but it is based on a simple concept, where each changed is registered against MD5 hash code, with relevant for CI/CD (not in our case) context information. I would agree that tinkering is not good idea, in this case there is only one entry and the schema had not been managed with Liquibase from the day. I am not going to insist but we could start with cleaner platform by doing so.